src/curated-corpus/components/actions/ScheduleCorpusItemAction/ScheduleCorpusItemAction.test.tsx

```diff
@@ -56,6 +57,10 @@ describe('The ScheduleCorpusItemAction', () => {
 
     mocks = [
       mock_AllScheduledSurfaces,
+      mock_scheduledItemsWithParams(
+        chosenDate.toFormat('yyyy-MM-dd'),
+        chosenSurfaceGuid
+      ),
```

Comment:
This is a mock for the Topic & Publisher info that was unreachable before in this form (???), but is needed now.